selfdrive/car/subaru/carstate.py

```diff
@@ -47,6 +50,11 @@ def update(self, cp, cp_cam, cp_body):
     ret.gearShifter = self.parse_gear_shifter(self.shifter_values.get(can_gear, None))
 
     ret.steeringAngleDeg = cp.vl["Steering_Torque"]["Steering_Angle"]
+    
+    if self.car_fingerprint in STEER_LIMITED_2020:
+      # have not found a steering rate message. calculate it manually
+      ret.steeringRateDeg = self.angle_rate_calulator.update(ret.steeringAngleDeg, cp.vl["Steering_Torque"]["COUNTER"])
```

Comment:
I would feel a bit more comfortable if we could detect a limit in the steering wheel angle delta over some time, since we haven't standardized calculating the steering wheel rate just yet, and any change this calculation would change fault workaround. Is there anything like > 10 deg in a second? we can use?

Reply:
we decided to allow erroneous detection to fix most faults